pkg/workloads/cortex/serve/serve.py

```diff
@@ -209,9 +209,24 @@ def start():
     spec = os.environ["CORTEX_API_SPEC"]
     project_dir = os.environ["CORTEX_PROJECT_DIR"]
     model_dir = os.getenv("CORTEX_MODEL_DIR", None)
-    tf_serving_port = os.getenv("CORTEX_TF_SERVING_PORT", None)
     storage = S3(bucket=os.environ["CORTEX_BUCKET"], region=os.environ["AWS_REGION"])
 
+    has_multiple_servers = os.getenv("CORTEX_MULTIPLE_TF_SERVERS", None)
+    if has_multiple_servers:
+        with FileLock("/run/used_ports.json.lock"):
```

Comment:
since `FileLock` has a file handle, do you think it would work to return it from `__enter__()`, use `with FileLock("/run/used_ports.json") as f:`, and remove line 217? Is that a good idea? Is that even possible?